Account/TokenServerClient.swift

```diff
@@ -31,15 +31,15 @@ public struct TokenServerToken {
                self.api_endpoint == other.api_endpoint
     }
 
-    public static func fromJSON(_ json: JSON) -> TokenServerToken? {
+    public static func fromJSON(_ jsonDict: [String: Any]) -> TokenServerToken? {
```

Comment:
Could you write unit tests for the `fromJSON` and for the `asJSON` methods?

Reply:
@lmarceau updated with unit tests for both methods, please have a look.